packages/teams-js/src/public/dialog.ts

```diff
@@ -21,7 +21,15 @@ import {
 import { runtime } from './runtime';
 
 /**
- * Namespace to interact with the dialog module-specific part of the SDK.
+ * This group of capabilities enables apps to show dialogs. There are two primary types of dialogs: URL-based dialogs and [Adaptive Card](https://learn.microsoft.com/adaptive-cards/) dialogs.
```

Comment:
Maybe introduce the modal dialog concept in the 1st sentence:

This group of capabilities enables apps to show **modal** dialogs.

Reply:
The experience of "dialogs" on mobile hosts is different than on web/Windows hosts. Specifically, the UX implementation isn't really a "dialog" per se -- e.g., it doesn't look like a pop-up box on top of the other content (which is what I typically think of as a dialog). On mobile, the implementation is basically another full screen element that looks like the app that launched it (e.g., the chrome/container is the same and not differentiated). The only thing that makes it remotely like a dialog is that it has an X in the upper left corner that, when tapped, will bring the user back to the underlying app. We can't change the "dialog" capability name, but I'm wondering if we should still use words like modal and dialog in the description?
It is still "modal" in the sense that user cannot do anything else in the host while it's showing but I didn't know if using the modal terminology was still appropriate here when there's no obvious "dialog" (even though I know that technically "modal" doesn't have to apply to a dialog)